src/SprykerEco/Zed/Adyen/Business/Hook/Mapper/MakePayment/AbstractMapper.php

```diff
@@ -127,6 +128,7 @@ protected function addFraudCheckData(
     ): AdyenApiRequestTransfer {
         $requestTransfer
             ->getMakePaymentRequest()
+            ->setShopperLocale('en_US')
```

Comment:
I don't think this is a correct way of setting customer locale.

Reply:
It was done for debugging.

I will remove this line of code. Because to provide current locale we should add a new dependency on the Locale module and it's not for bug fixing, but for improvement.

Reply:
Removed